src/_pytest/pytester.py

```diff
@@ -1529,7 +1525,7 @@ class Testdir:
 
     __test__ = False
 
-    CLOSE_STDIN = Pytester.CLOSE_STDIN
+    CLOSE_STDIN: "Final[type]" = Pytester.CLOSE_STDIN
```

Comment:
I *think* (haven't verified) that can be just the following and mypy will infer the reset:

```suggestion
    CLOSE_STDIN: "Final" = Pytester.CLOSE_STDIN
```

In which case, it'd be nice to add a `Final` for the two other constants below while you're at it.

Reply:
Thanks, this makes sense.